packages/backend-app-api/src/services/implementations/httpRouterService.ts

```diff
@@ -18,49 +18,33 @@ import {
   createServiceFactory,
   coreServices,
 } from '@backstage/backend-plugin-api';
-import Router from 'express-promise-router';
 import { Handler } from 'express';
-import { createServiceBuilder } from '@backstage/backend-common';
 
 /**
  * @public
  */
 export type HttpRouterFactoryOptions = {
   /**
-   * The plugin ID used for the index route. Defaults to 'app'
+   * The path prefix used for each plugin, defaults to `/api/`.
```

Comment:
I'd love for this to not have the slash at the end... For consistency. At least that's how I have always written things like this in the past :)

Reply:
If we auto insert a slash then we'd remove the ability to do stuff like `/plugin-catalog/`, but that might be worth it. Tbh I don't think this is something you want to touch particularly often so I feel it's alright if it needs a bit more precise configuration.

Reply:
Aight! As long as it's intentional. Might be a bit of a gotcha for adopters perhaps but not a big one. Just that we can't verify against mistakes.

Reply:
Could make it something like a `getPath` instead, bit more flexible and already an uncommon customization point

Reply:
Switched this to `getPath` instead